From 7e2232a7458a42e8d8096fbb682e5f374e4a694e Mon Sep 17 00:00:00 2001
From: Steven Fisher <steven.fisher@globalinvacom.com>
Date: Tue, 22 Oct 2019 14:30:17 -0400
Subject: [PATCH] contrib: use live555 version that is compatible with LGPL2

---
 contrib/src/live555/rules.mak | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/contrib/src/live555/rules.mak b/contrib/src/live555/rules.mak
index b68a77060b..7ebc3a1632 100644
--- a/contrib/src/live555/rules.mak
+++ b/contrib/src/live555/rules.mak
@@ -14,10 +14,8 @@ endif
 
 
 ifdef BUILD_NETWORK
-ifdef GNUV3
 PKGS += live555
 endif
-endif
 
 #ifeq ($(call need_pkg,"live555"),)
 #PKGS_FOUND += live555
@@ -58,7 +56,7 @@ endif
 
 live555: live555-$(LIVE555_HASH).tar.xz .sum-live555
 	rm -Rf live && $(UNPACK)
-	
+
 	mv live555-$(LIVE555_HASH) live
 
 	# Change permissions to patch and sed the source
@@ -107,7 +105,6 @@ endif
 SUBDIRS=groupsock liveMedia UsageEnvironment BasicUsageEnvironment
 
 .live555: live555
-	$(REQUIRE_GNUV3)
 	cd $< && for subdir in $(SUBDIRS); do \
 		echo "PREFIX = $(PREFIX)" >> $$subdir/Makefile.head && \
 		echo "LIBDIR = $(PREFIX)/lib" >> $$subdir/Makefile.head ; done
-- 
2.21.0 (Apple Git-122)

