From 03b8d54fc2b7e84a4e8a9c9a8835d166002368f3 Mon Sep 17 00:00:00 2001
From: Steven Fisher <steven.fisher@globalinvacom.com>
Date: Thu, 6 Dec 2018 18:59:37 -0500
Subject: [PATCH] contrib use live555 version that is compatible with LGPL

---
 contrib/src/live555/rules.mak | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/contrib/src/live555/rules.mak b/contrib/src/live555/rules.mak
index 5f2ec246f2..76bb4f566d 100644
--- a/contrib/src/live555/rules.mak
+++ b/contrib/src/live555/rules.mak
@@ -14,10 +14,8 @@ endif
 
 
 ifdef BUILD_NETWORK
-ifdef GNUV3
 PKGS += live555
 endif
-endif
 
 #ifeq ($(call need_pkg,"live555"),)
 #PKGS_FOUND += live555
@@ -57,7 +55,7 @@ LIVE_EXTRA_CFLAGS := $(EXTRA_CFLAGS) -fexceptions $(CFLAGS)
 
 live555: live555-$(LIVE555_HASH).tar.xz .sum-live555
 	rm -Rf live555 && $(UNPACK)
-	
+
 	mv live555-$(LIVE555_HASH) live
 
 	# Change permissions to patch and sed the source
@@ -104,7 +102,6 @@ endif
 SUBDIRS=groupsock liveMedia UsageEnvironment BasicUsageEnvironment
 
 .live555: live555
-	$(REQUIRE_GNUV3)
 	cd $< && for subdir in $(SUBDIRS); do \
 		echo "PREFIX = $(PREFIX)" >> $$subdir/Makefile.head && \
 		echo "LIBDIR = $(PREFIX)/lib" >> $$subdir/Makefile.head ; done
-- 
2.17.2 (Apple Git-113)

